internal/test/framework/framework.go

```diff
@@ -1,26 +1,16 @@
 package test
 
 import (
-	"flag"
```

Comment:
Why does dep updates have code changes? I will review the rest once this is resolved.